chart/README.md

```diff
@@ -10,4 +10,75 @@ This repository offers a convenience chart for the `kube-downscaler` [project](h
 | arguments          | list   | `[--interval=60,--include-resources=deployments,statefulsets,horizontalpodautoscalers,scaledobjects]` | Arguments to pass to the kube-downscaler binary |
 | excludedNamespaces | list   | `["namespace-a", "namespace-b"]`                                                                      | Namespaces to exclude from downscaling          |
 
-How the downscaler can be configured is described in the [kube-downscaler documentation](https://codeberg.org/hjacobs/kube-downscaler).
+
+
+# Deploy Kube-downscaler using Helm chart
+
+This directory contains tutorial to deploy Kube-downscaler.
+
+## Configuring your Deployment to downscale
+
+Please add below annotations based on timezone your deployment should run:
+```
+metadata:
+  annotations:
+    downscaler/uptime: "Mon-Fri 07:00-19:00 US/Eastern"
+```
+Note: For more configuration details please, refer [here](https://github.com/hjacobs/kube-downscaler#configuration).
+
+## Architecture
+The diagram below depicts how a Kube-downscaler agent control applications.
+![Alt text](images/architecture.png?raw=true "Kube Kube-downscaler diagram")
+
+## Quick Start
+Below are instructions to quickly install and configure Kube-downscaler.
+
+### Installing Kube-downscaler
+
+1. Make sure connected to right cluster:
+```
+kubectl config current-context
+```
+2. Set right environment depending on cluster:
+```
+export KDS_ENV='[minikube | testing | staging | production]'
+```
```

Comment:
This KDS_ENV variable as well as the `--values "config/${KDS_ENV}.yaml"` seem out of place? Is this a copy-paste error or am I missing something here? I din't find any config folder in the repository.

Reply:
yeah, thats deprecated